server/internal/infrastructure/mongo/mongodoc/pagination.go

```diff
@@ -0,0 +1,49 @@
+package mongodoc
+
+import (
+	"errors"
+
+	"github.com/reearth/reearthx/usecasex"
+)
+
+type Pagination struct {
```

Comment:
Use https://github.com/reearth/reearthx/blob/main/mongox/pagenation.go instead of reinventing the wheel.

Reply:
Done!